core/templates/dev/head/pages/signup-page/signup-page.controller.spec.ts

```diff
@@ -71,6 +71,7 @@ describe('Signup controller', function() {
           };
         }
       });
+      ctrl.$onInit();
```

Comment:
Explanation for this change: https://www.codelord.net/2017/01/09/unit-testing-angular-components-with-%24componentcontroller/